lib/beaker/host_prebuilt_steps.rb

```diff
@@ -13,7 +13,7 @@ module HostPrebuiltSteps
     SLEEPWAIT = 5
     TRIES = 5
     UNIX_PACKAGES = ['curl', 'ntpdate']
-    FREEBSD_PACKAGES = ['curl']
+    FREEBSD_PACKAGES = ['curl', 'perl5']
```

Comment:
Not sure how neccesary this is, as perl is included by default on all bsd builds apparently, but cant hurt (only takes a few secs to check it)